shell/lisa_shell

```diff
@@ -143,6 +143,20 @@ function _lisa-upgrade-pip {
     fi
 }
 
+function _lisa-install-nbextensions {
+    if which nodejs &>/dev/null; then
```

Comment:
should also check for "jupyter" command availability, since jupyterlab is an optional [notebook] dependency of LISA:
which jupyter &>/dev/null || return

Reply:
This ca be done... lemme try before a push without this just to be sure that the code words when we have only nodejs installed.

Reply:
Done